### ISSUE TITLE
Any advice on improving the glitchiness?

### ISSUE THREAD

**Issue by oijoijcoiejoijce** [LizhenWangT/StyleAvatar#9]
Thanks again for the great work. Unfortunately; the output is still quite glitchy as mentioned in the paper. Do you have any recommendations on how to improve that? 
For example: 
- Less head movement? 
- maybe rendering just the face instead of the whole head (i.e. keeping head movement of the source video constant and only rendering the face)

**Comment by alchemician** [comment]
same issue here, i tried face reenactment on the video in the drive link, here is a 5 second output. This clip is taken from 30 to 35 seconds from the original video


https://github.com/LizhenWangT/StyleAvatar/assets/93725624/bcf487e8-7ee2-4349-a5f8-e0027cb528a2

**Comment by LizhenWangT** [comment]
> same issue here, i tried face reenactment on the video in the drive link, here is a 5 second output. This clip is taken from 30 to 35 seconds from the original video
> 
> https://drive.google.com/file/d/1gQnw2XaGBjql6ok_IxD9MUihTfTqixT2/view?usp=sharing

This jitter exceeded my expectations. The results shown in my paper are based on C++. So I don't test the resutls on the python code. You can try to add a smooth term to the predicted faceverse parameters to improve the effect.

**Comment by LizhenWangT** [comment]
> Thanks again for the great work. Unfortunately; the output is still quite glitchy as mentioned in the paper. Do you have any recommendations on how to improve that? For example:
> 
> * Less head movement?
> * maybe rendering just the face instead of the whole head (i.e. keeping head movement of the source video constant and only rendering the face)

Yes, less head movement or only transferring the mouth-related parameters will improve the results.

**Comment by alchemician** [comment]
@LizhenWangT any ideas how to and which faceverse parameters to smooth?
